test/server/cards/02_SHD/events/Pillage.spec.ts

```diff
@@ -43,7 +44,30 @@ describe('Pillage', function() {
                     },
                     player2: {
                         hand: ['imperial-interceptor']
-                    }
+                    },
+                    autoSingleTarget: true
+                });
+
+                const { context } = contextRef;
+                const { player1, player2, pillage, imperialInterceptor } = context;
+
+                player1.clickCard(pillage);
+                player1.clickPrompt('Opponent');
+
+                expect(imperialInterceptor).toBeInZone('discard');
+
+                expect(player2).toBeActivePlayer();
+            });
+
+            it('should let the player target the opponent with autoSingleTarget false, so the opponent must select their last remaining card', function() {
```

Comment:
Instead of adding a new test here could we just remove autoSingleTarget from the above test and update it? Honestly just for testing the feature more broadly it would be good to remove all `autoSingleTarget` from this file and maybe one other, like Force Throw.

Reply:
Updated this unit test and force throw